sui_types/src/error.rs

```diff
@@ -316,6 +314,11 @@ pub enum SuiError {
     HkdfError(String),
     #[error("Signature key generation error: {0}")]
     SignatureKeyGenError(String),
+
+    // These are errors that occur when an RPC fails and is simply the utf8 message sent in a
+    // Tonic::Status
+    #[error("{0}")]
```

Comment:
I would sill want some prefix indicating an RPC issue?

Reply:
Yes...maybe?...Its complicated :D So a `SuiError` is stringified and shoved into a `tonic::Status`'s message field. Then we take that string and put it back here so prefixing it would have resulted in `RPC: Someotherprefix: error`. Although we can definitely play with this a bit more